### PR TITLE
Activity oracle sql error for favorites

### PR DESCRIPTION
**Pull request by nickvergessen**
can't really unit test this because the database table is missing.

Solution is taken from http://stackoverflow.com/a/18104929

Should backport the sql error (first commit) at least to 8.1 @karlitschek @DeepDiver1975 @cmonteroluque

### CODE REVIEW

**Comment by karlitschek** [comment]
:+1:

**Comment by nickvergessen** [comment]
http://sqlfiddle.com

Schema:
```sql
CREATE TABLE oc_activity (
  a_activity_id NUMBER(10) NULL,
  a_timestamp TIMESTAMP(0) NULL,
  a_priority NUMBER(10) NULL,
  a_type VARCHAR2(255) NULL,
  a_user VARCHAR2(64) NULL,
  a_affecteduser VARCHAR2(64) NULL,
  a_app VARCHAR2(255) NULL,
  a_subject VARCHAR2(255) NULL,
  a_subjectparams VARCHAR2(4000) NULL,
  a_message VARCHAR2(255) NULL,
  a_messageparams VARCHAR2(4000)  NULL,
  a_file VARCHAR2(4000)  NULL,
  a_link VARCHAR2(4000)  NULL,
  a_object_type VARCHAR2(255)  NULL,
  a_object_id NUMBER(10)  NULL
);
```

Queries
```sql
INSERT INTO oc_activity (a_app, a_type) VALUES ('ö', 'A');
INSERT INTO oc_activity (a_app, a_type) VALUES ('ö', 'a');
INSERT INTO oc_activity (a_app, a_type) VALUES ('Ü', 'A');
INSERT INTO oc_activity (a_app, a_type) VALUES ('Ü', 'a');

SELECT a.* FROM (
  SELECT * FROM oc_activity
  WHERE (
    ( 
      CASE 
      WHEN a_app <> 'ö' THEN 1
      WHEN a_app = 'ö' AND a_type = 'A' THEN 1
      ELSE 0
      END = 1
    )
  ) ORDER BY a_timestamp DESC) a WHERE ROWNUM <= 30;
```

But would be nice,  if someone can run it on a real DB as well.

**Comment by nickvergessen** [comment]
@DeepDiver1975 
```
ORA-01017: invalid username/password; logon denied Check environment: ORACLE_HOME=/usr/lib/oracle/xe/app/oracle/product/10.2.0/server ORACLE_SID=XE LD_LIBRARY_PATH=/usr/lib/oracle/xe/app/oracle/product/10.2.0/server/lib: NLS_LANG=AMERICAN_AMERICA.AL32UTF8 tnsnames.ora is readable
```

any idea?

**Comment by MorrisJobke** [comment]
@DeepDiver1975 Ping ;9

**Comment by nickvergessen** [comment]
Added execution of the query, so the unit tests now execute the query (with different table and column names, but the CASE is what we really want to test cross DB)

**Comment by nickvergessen** [comment]
@MorrisJobke @DeepDiver1975 please review this and also the backports:
8.2: https://github.com/owncloud/core/pull/20488
8.1: https://github.com/owncloud/core/pull/20489

**Comment by MorrisJobke** [comment]
I tested the favorite activities and they still work :+1:

**Comment by PVince81** [comment]
@DeepDiver1975 can you review and merge this Oracle stuff ? @MorrisJobke already tested

File: apps/files/lib/activity.php [hunk 1]
```diff
@@ -390,13 +391,15 @@ public function getQueryForFilter($filter) {
 			}
 			foreach ($favorites['folders'] as $favorite) {
 				$fileQueryList[] = '`file` LIKE ?';
-				$parameters[] = $favorite . '/%';
+				$parameters[] = \OC::$server->getDatabaseConnection()->escapeLikeParameter($favorite) . '/%';
```
Comment (DeepDiver1975) [inline apps/files/lib/activity.php]: cant we inject this?

File: apps/files/tests/activitytest.php [hunk 2]
```diff
@@ -333,6 +333,19 @@ public function testQueryForFilter($will, $query, $parameters) {
...
+		$this->executeQueryForFilter($result);
+	}
+
+	public function executeQueryForFilter(array $result) {
+		list($resultQuery, $resultParameters) = $result;
+		$resultQuery = str_replace('`file`', '`user`', $resultQuery);
+		$resultQuery = str_replace('`type`', '`key`', $resultQuery);
+
+		$connection = \OC::$server->getDatabaseConnection();
+		$result = $connection->executeQuery('SELECT * FROM `*PREFIX*privatedata` WHERE ' . $resultQuery, $resultParameters);
```
Comment (DeepDiver1975) [inline apps/files/tests/activitytest.php]: privatedata?

**Comment by DeepDiver1975** [comment]
@nickvergessen what about creating the table OC_activity on setup and drop the table on teardown?

this would allow us to really execute the query

**Comment by nickvergessen** [comment]
> @nickvergessen what about creating the table OC_activity on setup and drop the table on teardown?
this would allow us to really execute the query

Well I think it is not any different then replacing the column names and use an existing table. Since there is nothing making sure that the table we create in the test is the one the app creates on install.
Also the query is not really executed (just using different column names).

So not sure if it's worth the troubles

**Comment by nickvergessen** [comment]
Rebased and fixed the DI

**Comment by DeepDiver1975** [comment]
> Well I think it is not any different then replacing the column names and use an existing table. Since there is nothing making sure that the table we create in the test is the one the app creates on install.
> Also the query is not really executed (just using different column names).
> 
> So not sure if it's worth the troubles

Ah - so you are using the privatedata tables as replacement for the activities table.
Mind adding a comment? THX

:+1:

**Comment by MorrisJobke** [comment]
Woot?

```
05:29:56 1) OCA\Files_sharing\Tests\EtagPropagation::testOwnerWritesToSingleFileShare
05:29:56 Failed asserting that the etag for "" of user test-share-user2 has changed
05:29:56 Failed asserting that '8331f97f64dad95c58d377e47ce237a8' is not equal to <string:8331f97f64dad95c58d377e47ce237a8>.
05:29:56 
05:29:56 /ssd/jenkins/workspace/core-ci-linux/database/mysql/label/SLAVE/apps/files_sharing/tests/etagpropagation.php:143
05:29:56 /ssd/jenkins/workspace/core-ci-linux/database/mysql/label/SLAVE/apps/files_sharing/tests/etagpropagation.php:199
05:29:56 
```

**Comment by DeepDiver1975** [comment]
@nickvergessen can you please have a look at the failing test? THX

**Comment by nickvergessen** [comment]
It poped up when adding a comment, I will rebase once again, looks like a glitch to me

**Comment by DeepDiver1975** [comment]
````
09:51:55 There were 20 failures:
09:51:55 
09:51:55 1) OCA\Files\Tests\ActivityTest::testNotificationTypes
09:51:55 Your test case is not allowed to access the database.
09:51:55 
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:55 
09:51:55 2) OCA\Files\Tests\ActivityTest::testDefaultTypes
09:51:55 Your test case is not allowed to access the database.
09:51:55 
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:55 
09:51:55 3) OCA\Files\Tests\ActivityTest::testTranslate
09:51:55 Your test case is not allowed to access the database.
09:51:55 
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:55 
09:51:55 4) OCA\Files\Tests\ActivityTest::testGetSpecialParameterList
09:51:55 Your test case is not allowed to access the database.
09:51:55 
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:55 
09:51:55 5) OCA\Files\Tests\ActivityTest::testTypeIcon with data set #0 ('file_changed', 'icon-change')
09:51:55 Your test case is not allowed to access the database.
09:51:55 
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:55 
09:51:55 6) OCA\Files\Tests\ActivityTest::testTypeIcon with data set #1 ('file_created', 'icon-add-color')
09:51:55 Your test case is not allowed to access the database.
09:51:55 
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:55 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:55 
09:51:55 7) OCA\Files\Tests\ActivityTest::testTypeIcon with data set #2 ('file_deleted', 'icon-delete-color')
09:51:56 Your test case is not allowed to access the database.
09:51:56 
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:56 
09:51:56 8) OCA\Files\Tests\ActivityTest::testTypeIcon with data set #3 ('file_restored', false)
09:51:56 Your test case is not allowed to access the database.
09:51:56 
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:56 
09:51:56 9) OCA\Files\Tests\ActivityTest::testTypeIcon with data set #4 ('files_favorites', false)
09:51:56 Your test case is not allowed to access the database.
09:51:56 
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:56 
09:51:56 10) OCA\Files\Tests\ActivityTest::testTypeIcon with data set #5 ('unknown type', false)
09:51:56 Your test case is not allowed to access the database.
09:51:56 
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:56 
09:51:56 11) OCA\Files\Tests\ActivityTest::testGroupParameter
09:51:56 Your test case is not allowed to access the database.
09:51:56 
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:56 
09:51:56 12) OCA\Files\Tests\ActivityTest::testNavigation
09:51:56 Your test case is not allowed to access the database.
09:51:56 
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:56 
09:51:56 13) OCA\Files\Tests\ActivityTest::testIsFilterValid
09:51:56 Your test case is not allowed to access the database.
09:51:56 
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:56 
09:51:56 14) OCA\Files\Tests\ActivityTest::testFilterNotificationTypes with data set #0 ('files', array('NT0', 'file_created', 'file_changed', 'file_deleted', 'file_restored', 'files_favorites'), array('file_created', 'file_changed', 'file_deleted', 'file_restored'))
09:51:56 Your test case is not allowed to access the database.
09:51:56 
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:56 
09:51:56 15) OCA\Files\Tests\ActivityTest::testFilterNotificationTypes with data set #1 ('files', array('NT0', 'file_created', 'files_favorites'), array('file_created'))
09:51:56 Your test case is not allowed to access the database.
09:51:56 
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:56 
09:51:56 16) OCA\Files\Tests\ActivityTest::testFilterNotificationTypes with data set #2 ('files_favorites', array('NT0', 'file_created', 'file_changed', 'file_deleted', 'file_restored', 'files_favorites'), array('file_created', 'file_changed', 'file_deleted', 'file_restored'))
09:51:56 Your test case is not allowed to access the database.
09:51:56 
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:56 
09:51:56 17) OCA\Files\Tests\ActivityTest::testFilterNotificationTypes with data set #3 ('unknown filter', array('NT0', 'file_created', 'file_changed', 'file_deleted', 'file_restored', 'files_favorites'), false)
09:51:56 Your test case is not allowed to access the database.
09:51:56 
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:56 
09:51:56 18) OCA\Files\Tests\ActivityTest::testQueryForFilter with data set #0 (RuntimeException Object (...), '`app` = ?', array('files'))
09:51:56 Your test case is not allowed to access the database.
09:51:56 
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:56 
09:51:56 19) OCA\Files\Tests\ActivityTest::testQueryForFilter with data set #1 (array(array(), array()), ' CASE WHEN `app` <> ? THEN 1 ...D = 1 ', array('files', 'files', 'file_created', 'file_changed'))
09:51:56 Your test case is not allowed to access the database.
09:51:56 
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98
09:51:56 
09:51:56 20) OCA\Files\Tests\ActivityTest::testQueryForFilter with data set #2 (array(array('file.txt', 'folder'), array('folder')), ' CASE WHEN `app` <> ? THEN 1 ...D = 1 ', array('files', 'files', 'file_created', 'file_changed', 'file.txt', 'folder', 'folder/%'))
09:51:56 Your test case is not allowed to access the database.
09:51:56 
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/tests/lib/testcase.php:60
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/3rdparty/pimple/pimple/src/Pimple/Container.php:113
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/appframework/utility/simplecontainer.php:104
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/lib/private/server.php:792
09:51:56 /ssd/jenkins/workspace/server-master-linux-php7-ci/database/sqlite/label/SLAVE/apps/files/tests/activitytest.php:98

**Pull request merged**